Makefile

```diff
@@ -140,6 +144,16 @@ generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs Go related generate tar
 		--input-dirs=./api/v1alpha2 \
 		--output-file-base=zz_generated.conversion \
 		--go-header-file=./hack/boilerplate/boilerplate.generatego.txt
+	$(CONTROLLER_GEN) \
+        object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+        paths=./cmd/clusterctl/api/...
+	## generate code for embedded clusterctl api manifest
```

Comment:
Let's create another target here instead of adding to `generate-go`? Maybe something like `generate-clusterctl` or `generate-bindata`?

It's fine if we want to call the new target from here, but I'd rather have it contained in a single target for future maintenance tasks

Reply:
+1